specificatie/klanten/openapi.yaml

```diff
@@ -219,6 +219,13 @@ paths:
         required: false
         schema:
           type: integer
+      - name: includeNullValues
+        in: query
+        description: Bepaald of Null waarden opgenomen moeten worden in de response.
+        required: false
+        schema:
+            type: boolean
+            default: false
```

Comment:
Ik zou de default liever op `true` willen hebben. Immers het teruggeven van null waarden geeft de ontwikkelaar van de consumer applicatie veel meer inzicht en voorkomt fouten. Pas als alles is uitgetest kun je de null waarden weglaten om de performance te verbeteren als dat nodig is.

Reply:
Ik heb dit nog niet aangepast omdat het terug willen ontvangen van attributen met null waarden voor mij nou juist de uitzonderingssituatie is. De situatie die je straks in productie wil geïmplementeerd wil hebben, die waarbij je geen attributen met null waarden ontvangt, zou n.m.m. juist de default moeten zijn.

BTW: ook de naam 'includeNullValues' is wat mij betreft nog niet in beton gegoten. Als er betere ideeën zijn dan hoor ik dat graag.